### DISCUSSION TITLE
How to host learnr tutorial in shiny server with Azure VM

### DISCUSSION THREAD

**Discussion by micheal-arun** [rstudio/learnr#754] [Q&A]
Hi, 

I am trying to host one of the assignment for my class in shiny server with Azure virtual machine set up. but I am getting the following error. the same file I could run via R studio server. what am I missing here?

![image](https://user-images.githubusercontent.com/116641065/202083333-5df6ad94-d6ff-44d8-bfe3-bc146f7bb06f.png)

**Comment by gadenbuie** [comment]
I'm sorry, I don't know much about Azure. From the error message, though, it seems that the app needs to be able to write to `/srv/shiny-server/mid_term` but can't. I'd browse the [Shiny Server Admin Guide](https://rstudio.github.io/shiny-server/os/0.4.0/#hosting-model) for some help and check the permissions of `/srv/shiny-server` to make sure that the shiny server user has the right permissions for that path.
